STM32/DC/Core/Src/DCBoard.c

```diff
@@ -129,7 +153,7 @@ static void printResult(int16_t *pBuffer, int noOfChannels, int noOfSamples)
         return;
     }
```

Comment:
On line 152 I think the format specifier should be changed from `0x%x` to `0x%08x` for consistency.